projects/canopy/src/lib/forms/sort-code/sort-code.component.ts

```diff
@@ -47,6 +47,7 @@ export class LgSortCodeComponent implements OnInit, ControlValueAccessor {
   valueChanges: Subscription;
   hasError: boolean;
 
+  @Input() invalidSortCode$: Subject<void>;
```

Comment:
Just wondering, does this input get used in the template? Could you add an example to the Story notes? We also have a table in the story notes which describes each input, so it would be great to have another entry there for this one.

Reply:
Yes, this input gets used in the template as well :). I will take a look at adding an example in story notes as well as documenting this input. Thanks Owen 👍

Reply:
Great thanks @Zimann! Documentation is something we're really keen to improve on, so I think it's worth taking extra care with 🙂